source/renderer/dialog/preferences.js

```diff
@@ -126,8 +130,8 @@ class PreferencesDialog extends ZettlrDialog {
         // download no non-available language is set.
         language.value = global.config.get('appLang')
         // ... and beneath the select
-        $('#app-lang-download-indicator').text(langLocalisation)
-        $('#app-lang-download-indicator').append(this._spinner)
+        this.downloadIndicatorElement.textContent = langLocalisation
+        this.downloadIndicatorElement.append(this._spinner)
```

Comment:
[append()](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/append) is now part of the native API. 🎉  It was `appendChild` only before as you said.

I'll double check the spinner and report back.

Reply:
Thanks!

Reply:
So, I don't think this spinner element works, because the necessary Core UI CSS is not loaded. I checked both `package.json` files but I can't see it as a dependency there.

Reply:
Just to clarify, the Javascript part of the code works and the element is placed where it supposed to go, but it is invisible due to the missing CSS.